app/assets/stylesheets/application.scss

```diff
@@ -14,10 +14,25 @@
  // Custom bootstrap variables must be set or imported *before* bootstrap.
  @import "bootstrap";
 
+ .jumbotron {
+ background-image: url("https://res.cloudinary.com/sanitarium/image/fetch/q_auto/https://www.sanitarium.com.au/getmedia%2Fae51f174-984f-4a70-ad3d-3f6b517b6da1%2Ffruits-vegetables-healthy-fats.jpg%3Fwidth%3D1180%26height%3D524%26ext%3D.jpg");
+ background-size: cover;
+ color: white;
+ margin-top: 0px;
+ margin-bottom: 0px;
+ font-size: 60px
+}
+
+.header-opacity {
```

Comment:
Nice job with the black transparency.  I couldn't get mine to look right and I think it was because I was doing a gray transparent